libs/deal-ticket/src/components/deal-ticket/deal-ticket-fee-details.tsx

```diff
@@ -267,19 +267,7 @@ export const DealTicketFeeDetails = ({
         ? liquidationEstimateBestCase
         : liquidationEstimateWorstCase
       ).toString(),
-      assetDecimals
-    );
-    liquidationPriceEstimateFormatted = formatRange(
-      (liquidationEstimateBestCase < liquidationEstimateWorstCase
-        ? liquidationEstimateBestCase
-        : liquidationEstimateWorstCase
-      ).toString(),
-      (liquidationEstimateBestCase > liquidationEstimateWorstCase
-        ? liquidationEstimateBestCase
-        : liquidationEstimateWorstCase
-      ).toString(),
-      assetDecimals,
-      quantum
+      marketDecimals
```

Comment:
liquidation price should use assetDecimals, when using market decimals we are getting wrong values
<img width="1185" alt="image" src="https://github.com/vegaprotocol/frontend-monorepo/assets/1754247/7c64a088-6d96-4a50-9c6d-22c97a1ab3ee">

Reply:
this might be a problem. is it a bug?
as far as I know from @barnabee `liquidation price` and `notional` are dependent on quote, and not settlement asset. so we should use market dp instead. 

why liquidation price uses asset dp instead?

Reply:
It uses asset decimals and it has fractional part also.
<img width="380" alt="image" src="https://github.com/vegaprotocol/frontend-monorepo/assets/1754247/be467cd3-e0d3-456d-8262-29aad99bbe38">

@witgaw @barnabee should we change it in API to use market decimal places?